### ISSUE TITLE
The inspector should stop deprecations logging to the console

### ISSUE THREAD

**Issue by teddyzeenny** [emberjs/ember-inspector#345]
In case of deprecations just warn once: "Deprecations were detected, see the Ember Inspector deprecations tab for more details."

Should Ember core disable the deprecation logs by default and only warn once (except when a flag is set), or should that only happen when the inspector is open?

cc @rwjblue

**Comment by rwjblue** [comment]
I think that it should only be silenced if the inspector is open (I don't want these to have an oppurtunity of being missed).

**Comment by tomdale** [comment]
@rwjblue It seems OK to emit a message that deprecations are logged in the inspector and users should look there if they are interested.

**Comment by rwjblue** [comment]
@tomdale - Yes, that is perfect when the inspector is open/available.  If I open an app in only Internet Explorer (without using the inspector bookmarklet) or Chrome/FireFox without the inspector installed and open, I should still see deprecations.

**Comment by maheshsenni** [comment]
I was able to stop the deprecation logging easily. To show the warning once, I depend on the length of the `deprecations` array. This prints the warning again if the deprecations are cleared inside the inspector. Thinking of a better approach to handle this problem.

**Comment by teddyzeenny** [comment]
@maheshsenni maybe you can set a flag after the first warning:

```js
if (!this._warned) {
  self.get('adapter').debug("Deprecations were detected, see the Ember Inspector deprecations tab for more details.");
  this._warned = true;
}
``

**Comment by maheshsenni** [comment]
Submitted a pull request for this #347

**Comment by teddyzeenny** [comment]
Fixed by https://github.com/emberjs/ember-inspector/pull/347
